glam/api/views.py

```diff
@@ -84,9 +80,14 @@ def aggregations(request):
             "Missing required query parameters: {}".format(", ".join(sorted(missing)))
         )
 
+    # If release channel, make sure the user is authenticated.
```

Comment:
this will have to get rebased on master since this view has changed a bit, but I don't think it'll be too difficult